### PR TITLE
Add example for :class:`Point`

### PR DESCRIPTION
**Pull request by jojo96**
Added example for point class

## Changelog / Overview

Added example for Point class

## Motivation

More examples make it easier for new users
## Explanation for Changes

Added more examples for easier usage as there were no previous examples for Point class

[Documentation Reference](https://manimce--####.org.readthedocs.build)

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [x] The PR title is descriptive enough
- [x] The PR is labeled correctly
- [x] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Review by Darylgolden — DISMISSED** [review]
Can you use the `!mdocstring` command of the bot to format your code? The indentation is wrong, which is why the doc build failed.

**Comment by jojo96** [comment]
> Can you use the `!mdocstring` command of the bot to format your code? The indentation is wrong, which is why the doc build failed.

Hi, @Darylgolden I have edited the example and now all the tests have passed.

File: manim/mobject/types/point_cloud_mobject.py [hunk 1]
```diff
@@ -271,6 +271,25 @@ def generate_points(self):
...
+    Examples
+    --------
+    .. manim:: ExamplePoint
+        :save_last_frame:
+
+        class ExamplePoint(ThreeDScene):
+            def construct(self):
+                colorList = [RED, GREEN, BLUE, YELLOW]
+                for i in range(200):
+                    point = Point(location=[0.63*np.random.randint(-4, 4), 0.37*np.random.randint(-4, 4),0], color=np.random.choice(colorList))
```
Comment (naveen521kk) [inline manim/mobject/types/point_cloud_mobject.py:285]: Again use the bot's `!md` command.

File: manim/mobject/types/point_cloud_mobject.py [hunk 2]
```diff
@@ -271,6 +271,26 @@ def generate_points(self):
...
 class Point(PMobject):
+    """A point.
+
+    Examples
+    --------
+
+    .. manim:: ExamplePoint
+        :save_last_frame:
+
+        class ExamplePoint(ThreeDScene):
```
Comment (PhotonSpheres) [inline manim/mobject/types/point_cloud_mobject.py:282]: Just curious, is there a specific reason you chose `ThreeDScene` over `Scene`?

Comment (jojo96) [reply manim/mobject/types/point_cloud_mobject.py:282]: It works with Scene too.
But while declaring a new Point, I have to pass a 3d point like location=[0,0,0]. If I pass [0,0], it does not work. So, I thought to go with ThreeDScene.

Comment (PhotonSpheres) [reply manim/mobject/types/point_cloud_mobject.py:282]: I see. IMHO `ThreeDScene` is not really necessary here. Maybe just put coordinates like [x,y,0] (every coordinates that correspond to a scene are always 3d).

Comment (jojo96) [reply manim/mobject/types/point_cloud_mobject.py:282]: Ok sure, will just use Scene.

File: manim/mobject/types/point_cloud_mobject.py [hunk 3]
```diff
@@ -271,6 +271,26 @@ def generate_points(self):
 
 
 class Point(PMobject):
+    """A point.
```
Comment (naveen521kk) [inline manim/mobject/types/point_cloud_mobject.py:274]: I don't think is helpful. Can you remove this or add some more lines explaining what this does?

Comment (jojo96) [reply manim/mobject/types/point_cloud_mobject.py]: Removed this

**Review by naveen521kk — APPROVED** [review]

**Pull request merged**